src/app/components/ContactToolbar.js

```diff
@@ -16,6 +24,16 @@ const ContactToolbar = ({ user, onCheck, onDelete, checked = false, activeIDs =
         }, []);
     }, [activeIDs, contactEmailsMap]);
 
+    if (simplified) {
+        return (
+            <div className="toolbar flex noprint">
+                <Link to="/contacts" className="toolbar-button">
```

Comment:
You should use `Link` from `react-router-dom` because you don't need to manage external link here.

Reply:
Oh, I thought the idea was to always use `Link` from `react-components` no matter what, so that the component would decide what to do. But I can change it if that's not the plan